BUILD.gn

```diff
@@ -63,10 +63,19 @@ if (is_win) {
   # another tool, Zucchini, as a drop-in replacement.
   if (build_delta_installer) {
     copy("copy_zucchini_binary_as_courgette") {
+      zucchini_toolchain = ""
+      if (target_cpu == "x86") {
+        # 32 bit Zucchini runs into memory allocation errors when virtual memory
+        # utilization is high. Using the 64 bit version fixes this.
+        zucchini_toolchain = "(//build/toolchain/win:win_clang_x64)"
+      }
+      zucchini_dep = "//components/zucchini:zucchini" + zucchini_toolchain
```

Comment:
Just curious - `//components/zucchini:zucchini(//build/toolchain/win:win_clang_x64)` is right target name?

Reply:
Yes. For comparison, see https://github.com/brave/brave-core/blob/28ba6f300e440629f3315084d79b86755cb37ac1/app/win/BUILD.gn#L19.

Reply:
Oh, it's gn's way for specifying specific toolchain? I didn't know that!

Reply:
Yes! It's pretty cool I think :)

Reply:
Wow, I found it - https://gn.googlesource.com/gn/+/HEAD/docs/reference.md#example-of-cross_toolchain-dependencies 👍🏼